deps/Makefile

```diff
@@ -670,6 +670,7 @@ $(eval $(call LLVM_PATCH,llvm-3.7.1))
 else ifeq ($(LLVM_VER),3.7.1)
 $(eval $(call LLVM_PATCH,llvm-3.7.1))
 $(eval $(call LLVM_PATCH,llvm-3.7.1_2))
+$(eval $(call LLVM_PATCH,llvm-3.7.1_3))
 $(LLVM_SRC_DIR)/llvm-3.7.1_2.patch-applied: $(LLVM_SRC_DIR)/llvm-3.7.1.patch-applied
```

Comment:
I'd don't have enough Makefile-fu to understand why this line exists. So, I didn't know if I should update the target to `$(LLVM_SRC_DIR)/llvm-3.7.1_3.patch-applied`.